### PR TITLE
Small fixes to the improved chart

### PR DESCRIPTION
**Pull request by manuelwedler**
Follow-up fixes to https://github.com/sourcifyeth/ui/pull/1#issuecomment-2376733446

## Summary by CodeRabbit

- **New Features**
	- Introduced a new optional `transparent` property in the ChainSelect component for customizable styling.
	- Updated the display of options in the ChainSelect component for improved clarity.
	- Enhanced the ChartSection by splitting the total contracts verified into two distinct headings for better readability.

- **Bug Fixes**
	- Adjusted the order of properties used in the ChainSelect options to ensure accurate representation of names.
  
- **Style**
	- Added styles for a transparent select input in the ChainSelect component.

### CODE REVIEW

**Review by kuzdogan — CHANGES_REQUESTED** [review]
How does this filter the sourcify-chains

File: src/components/ChainSelect/index.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default function ChainSelect({
       onChange={handleChainIdChange}
       value={value}
       options={filteredChains.map((chain) => ({
-        name: `${chain.title || chain.name} (${chain.chainId}) `,
+        name: `${chain.name || chain.title} (${chain.chainId}) `,
```
[the remark this answers is not in the document]
Comment (manuelwedler) [reply src/components/ChainSelect/index.tsx:37]: Yes we have that many :D over 200 including the "supported: false" ones

I still think it is better to use the name in the selector here. The title is just too long and lets some labels overflow

[the remark this answers is not in the document]
Comment (kuzdogan) [reply src/components/ChainSelect/index.tsx:37]: Except the weird `title` for chain 1339, I find titles to be more descriptive. E.g. Amoy and Mumbai do not have "Polygon" in their names, if you were to search with names. Instead should we make the dropbox wider and wrap the lines in the dropbox menu?

[the remark this answers is not in the document]
Comment (manuelwedler) [reply src/components/ChainSelect/index.tsx]: Alright changed it back to title and fixed with hiding the overflow and increasing height a little. 1339 is actually the only one affected by this.

**Review by kuzdogan — APPROVED** [review]

**Pull request merged**